### ISSUE TITLE
Add Intellivision emulation

### ISSUE THREAD

**Issue by LeRiton** [digitalLumberjack/recalbox-os#185]
Because BurgerTime.

Bliss seems to be the way to go (and also unmaintained?).

No LibRetro port.
[Bliss for OpenEmu](https://github.com/OpenEmu/Bliss-Core)

Thanks for the fantastic job, realy enjoying my RecalBox!

**Comment by salquem** [comment]
+1, would be fantastic !

**Comment by kryptto** [comment]
I would love too play Astrosmash, Utopia.....

**Comment by reggaemanu** [comment]
Should be better to go with MESS, it's already available with mame source (just a flag to add at compilation time to compile mess instead of mame) and support intellivision.
I think that's what they are using in retropie for intellivision (although I didn't checked)

**Comment by ziviani** [comment]
Unfortunately just a few people will like to have Intellivision's emulation on RecalboxOS, because nobody shown interest since last message sent, one year ago

**Comment by kryptto** [comment]
Not true, I just think there is no reason to continue posting over and over the support for this platform.

Sent from my iPad

On Mar 24, 2016, at 10:06 PM, ziviani <notifications@github.com<mailto:notifications@github.com>> wrote:


Unfortunately just a few people will like to have Intellivision's emulation on RecalboxOS, because nobody shown interest since last message sent, one year ago

**Comment by substring** [comment]
as a reminder for lazy buddies : https://github.com/recalbox/recalbox-os/issues/564#issue-124281603

**Comment by mtmello2003** [comment]
Just to let all of you know: +1 would love to play burgtime and astromash from mattel intelivision. Retropie support it, why not Recalbox? :)

**Comment by cedr-x** [comment]
Burgertime, beauty and the beast, soccer, frogbog, Tron deadly discs, motocross, ... were my favorites. I'd love recalbox to support it. But thanks by the way. Videopac is already fantastic
